mitmproxy/addons/dumper.py

```diff
@@ -124,6 +125,9 @@ def _echo_request_line(self, flow):
             url = flow.request.pretty_url
         else:
             url = flow.request.url
+        terminalWidthLimit = shutil.get_terminal_size()[0] - 25
```

Comment:
Oh yes. I missed that case. I subtracted 25 because I thought the idea was to fit the output into one line. We already have `GET`, `HTTP/x.x` and ` PUSH_PROMISE`, so in the worst case we will need 25 places to fit them all into the line.  

I think it would be better to remove the `- 25` and allow it to occupy two lines in some cases. 
Or maybe `terminalWidthLimit = shutil.get_terminal_size()[0] // 2`